lib/fathead/nose2/parse.rb

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env ruby
+require 'nokogiri'
+require 'open-uri'
+
+NOSE_HOME = "http://nose2.readthedocs.io/en/latest/getting_started.html"
+topics = ARGV
+
+topics.each do |topic|
+  url = "http://nose2.readthedocs.io/en/latest/getting_started.html##{topic}"
+  doc = Nokogiri::HTML(open(url))
+  abstract = doc.css("##{topic}").css("p").to_s
+  abstract = abstract.gsub("\n", " ").gsub(/class\=\S{1,2}[a-z]*(\s[a-z]*)?\S{1,2}/, "")
```

Comment:
@rnkaufman You'll need to keep `<span>` and `<a>` tags out of the abstract. 

Code snippets should be wrapped in `<pre><code>`. Text content should be wrapped in `<p>`

The whole abstract needs to be wrapped in `<section class="prog__container"></section>`

Checkout the Fathead Docs for examples 👍